src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -123,11 +129,11 @@ def upload_access_results(
     access_result: Dict[str, List[Row]],
     dataset_graph: DatasetGraph,
     privacy_request: PrivacyRequest,
-) -> None:
+) -> List[str]:
     """Process the data uploads after the access portion of the privacy request has completed"""
+    download_urls: List[str] = []
```

Comment:
It's possible for multiple rules in policy to each point to an s3 upload location (and in the future, may include other non-s3 upload locations), so we support returning multiple download links